internal/bminventory/inventory.go

```diff
@@ -1844,23 +1844,15 @@ func (b *bareMetalInventory) updateNonDhcpNetworkParams(updates map[string]inter
 		return common.NewApiError(http.StatusBadRequest, err)
 	}
 
-	var err error
-	*machineCidr, err = network.CalculateMachineNetworkCIDR(apiVip, ingressVip, cluster.Hosts)
```

Comment:
Not sure if you can remove it, in none-dhcp mode users will not set machine-network-cidr anyway, so backend will set it according to AIPVip and IngressVIP no?